intranet/apps/eighth/views/activities.py

```diff
@@ -47,6 +47,28 @@ def activity_view(request, activity_id=None):
     return render(request, "eighth/activity.html", context)
 
 
+@login_required
+def eighth_location(request):
```

Comment:
I'm not 100% sure if `activities.py` is the right place for this view. Perhaps `signup.py` would be better?